packages/miniflare/src/plugins/assets/index.ts

```diff
@@ -278,7 +294,10 @@ const createReverseMap = async (dir: string) => {
 				return;
 			} else {
 				const pathHash = bytesToHex(
-					await hashPath(encodeFilePath(relativeFilepath, path.sep))
+					await hashPath(
+						encodeFilePath(relativeFilepath, path.sep) +
+							filestat.mtimeMs.toString()
```

Comment:
don't _love_ needing to keep this in sync with the other thing above, but it's fine for now.